src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxdatainterface.py

```diff
@@ -54,38 +53,58 @@ def _source_data_to_extractor_kwargs(self, source_data: dict) -> dict:
     @validate_call
     def __init__(
         self,
-        file_path: FilePath,
-        verbose: bool = True,
+        folder_path: Optional[DirectoryPath] = None,
+        stream_id: Optional[str] = None,
         es_key: Optional[str] = None,
+        verbose: bool = True,
+        file_path: Optional[FilePath] = None,
```

Comment:
I would like to deprecate this but I don't want this discussion to get in the way of this PR. If people who review this agree we could start a soft deprecation as we usually do.

Reminder: Spiekinterface takes folder path as input (as in this PR) and not file_path as we currently do in main.